frontend/src/entities/configuration/activated-services-configuration.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import {
+    Alert,
+    Container,
+    ContentLayout,
+    Grid,
+    Header,
+    SpaceBetween,
+    Toggle,
+    Box
+} from '@cloudscape-design/components';
+import React from 'react';
+import {ValidationFormActionMethod} from '../../shared/validation';
+
+const configurableServices = {
+    batchTranslate: 'Amazon Translate batch',
+    realtimeTranslate: 'Amazon Translate real-time',
+    emrCluster: 'Amazon EMR',
+    labelingJob: 'Amazon Ground Truth'
+};
+
+export type ActivatedServicesConfigurationProps = {
+    setFields: (values: { [key: string]: any }, method?: ValidationFormActionMethod) => void;
```

Comment:
This is the right signature, and it makes no difference but I'd consider just importing `SetFieldsFunction` and having `setFields: SetFieldsFunction` so it is cleaner. But not 100% necessary.